src/main/java/com/product/manager/service/impl/LoggingServicerImpl.java

```diff
@@ -0,0 +1,62 @@
+package com.product.manager.service.impl;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.product.manager.filter.CustomURLFilter;
+import com.product.manager.service.LoggingService;
+
+@Service
+public class LoggingServicerImpl implements LoggingService {
```

Comment:
Anh chua hieu lam y tuong cua em cho nay. Sao minh lai can loggingService nhi?
Viec logging la Cross-Cutting concern , no khong phai la business cua app nen khong nen de no la Service. Co the implement truc tiep o noi ma em thuc hien log ay

Reply:
Dạ ở service này em tự custom lại log để show ra thông tin từ request ạ